stubs/ldap3/ldap3/protocol/rfc4511.pyi

```diff
@@ -1,32 +1,26 @@
-from typing import Any as _Any
-
-# Enable when pyasn1 gets stubs:
-# from pyasn1.type.univ import Boolean, Choice, Enumerated, Integer, Null, OctetString, Sequence, SequenceOf, SetOf
-Boolean = _Any
-Choice = _Any
-Enumerated = _Any
-Integer = _Any
-Null = _Any
-OctetString = _Any
-Sequence = _Any
-SequenceOf = _Any
-SetOf = _Any
-
-LDAP_MAX_INT: int
-MAXINT: _Any
-rangeInt0ToMaxConstraint: _Any
-rangeInt1To127Constraint: _Any
-size1ToMaxConstraint: _Any
-responseValueConstraint: _Any
-numericOIDConstraint: _Any
-distinguishedNameConstraint: _Any
-nameComponentConstraint: _Any
-attributeDescriptionConstraint: _Any
-uriConstraint: _Any
-attributeSelectorConstraint: _Any
+from typing_extensions import Final as _Final
+
+from pyasn1.type.constraint import ConstraintsIntersection, SingleValueConstraint, ValueRangeConstraint, ValueSizeConstraint
+from pyasn1.type.namedtype import NamedTypes
+from pyasn1.type.namedval import NamedValues
+from pyasn1.type.tag import TagSet
+from pyasn1.type.univ import Boolean, Choice, Enumerated, Integer, Null, OctetString, Sequence, SequenceOf, SetOf
+
+LDAP_MAX_INT: _Final = 2147483647
```

Comment:
If I understood https://github.com/python/typeshed/issues/7258#issuecomment-1369821606 correctly

Reply:
This looks fine to me, but I don't see any reason to do the `from typing_extensions import Final as _Final` thing -- we can just do `from typing_extensions import Final` :)

Reply:
The class `Final` is declared lower down in this file. Should be more obvious now that there's a comment though